docs/data_management/transfer/tutorial/globus_group_management.md

```diff
@@ -0,0 +1,273 @@
+
+If you are new to Globus, we recommend starting with our [Globus Tutorials for Individual Researchers](./globus_individual_tutorial.md) first to familiarize yourself with how Globus is used.
+
+Below is a list of topics covered in this article.
+
+- General information:
+    - [Where Can I Find the Groups Page in the Globus Web App?](#where-can-i-find-the-groups-page-in-the-globus-web-app)
+    - [How Do I Find a Group?](#how-do-i-find-a-group)
+- For Group members:
+    - [How Do I Request to Join a Group?](#how-do-i-request-to-join-a-group)
+    - [How Do I Find and Accept an Invitation From a Group?](#how-do-i-find-and-accept-an-invitation-from-a-group)
+    - [How Do I Leave a Group?](#how-do-i-leave-a-group)
+- For Group administrators:
+    - Managing Groups and Subgroups
+        - [How Do I Create a Group?](#how-do-i-create-a-group)
+        - [How Do I Create a Subgroup?](#how-do-i-create-a-subgroup)
+        - [How Do I See Subgroups of a Group?](#how-do-i-see-subgroups-of-a-group)
+        - [How Do I Modify a Group or Subgroup?](#how-do-i-modify-a-group-or-subgroup)
+            - [How Do I Make My Group Invite Only?](#how-do-i-make-a-group-invite-only)
+            - [How Do I Hide My Group from Others?](#how-do-i-hide-a-group-from-others)
+            - [How Do I Keep Members from Seeing Each Other?](#how-do-i-keep-group-members-from-seeing-each-other)
+        - [How Do I Delete a Group or Subgroup?](#how-do-i-delete-a-group-or-subgroup)
+    - Managing Group Members and Roles
+        - [How Do I Invite Others to a Group?](#how-do-i-invite-others-to-a-group)
+        - [How Do I See Group Members?](#how-do-i-see-group-members)
+        - [How Do I Modify Group Members?](#how-do-i-modify-group-members)
+            - [How Do I Change a Group Member's Role Level?](#how-do-i-change-a-group-members-role-level)
+            - [What Does Each Role Mean?](#what-does-each-role-mean)
+            - [How Do I Remove a Group Member?](#how-do-i-remove-a-group-member)
+
+## Where Can I Find the Groups Page in the Globus Web App?
+
+1. [Get onto the Globus Web App](./globus_individual_tutorial.md#how-do-i-get-onto-the-globus-web-app).
+1. In the left-hand navigation panel, click Groups to be taken to the Groups page. All Group management activities in this tutorial document start on this page, so take some time to familiarize yourself with getting to this point.
+
+    ![Groups page of the Globus Web App.](./images/common/groups.png)
+
+## How Do I Find a Group?
+
+1. [Get to the Groups page of the Globus Web App](#where-can-i-find-the-groups-page-in-the-globus-web-app).
+1. If you are looking for new groups, ensure the "MY GROUPS" Checkbox is unchecked. If you are looking for groups you are already a member of, ensure the "MY GROUPS" Checkbox is checked.
+
+    ![My groups checkbox unchecked.](./images/common/groups-checked.png)
```

Comment:
Would be good if you could  mark or add an arrow pointing to the 'My Groups' button in the screenshot.

Reply:
Go this one and the unchecked version.